packages/core/src/scxml.ts

```diff
@@ -318,12 +318,6 @@ function toConfig(
         target: target ? `#${target}` : undefined
       };
     }
-    case 'final': {
```

Comment:
Hm, *I think* this change will allow (and create) more invalid configs than previously. `<final>` accepts way fewer children element types than regular `<state>`. Also - from what I see (but I could have missed something) we don't yet support `<donedata>` conversion - which is `<final>`-specific, so it would make sense to me to keep a dedicated `<final>` converter (instead of reusing the looser~ converter)

Reply:
I'm going to refactor this in a separate PR - this file needs to be cleaned up in general. Main goal is getting the tests to pass for now.